aptos-move/mvhashmap/src/versioned_group_data.rs

```diff
@@ -223,9 +261,10 @@ impl<
     /// Mark all entry from transaction 'txn_idx' at access path 'key' as an estimated write
     /// (for future incarnation). Will panic if the entry is not in the data-structure.
     pub fn mark_estimate(&self, group_key: &K, txn_idx: TxnIndex, tags: HashSet<T>) {
```

Comment:
pass & to tags, and make sure executor does not clone but key and tags before calling the method.